packages/@okta/vuepress-site/docs/reference/api/xaas/index.md

```diff
@@ -0,0 +1,395 @@
+
+##### Request
+
+```bash
+curl -v -X POST \
+???
+```
+
+##### Response
+
+```json
+{
+???
+}
+```
+
+### Retrieve an Import Session
+
+<ApiOperation method="get" url="/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}" />
+
+Retrieves an Import Session by its ID
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `identitySourceId`  | String | The ID obtained from creating a Customer Identity Source integration in Okta |
+| `sessionId`  | String | The ID for the Import Session to retrieve |
+
+#### Response body
+
+The requested ??? object
+
+#### Use example
+
+This request...:
+
+##### Request
+
+```bash
+curl -v -X GET \
+???
+```
+
+##### Response
+
+```json
+{
+  ???
+} 
+```
+
+##### Error response example
+
+Here you can show any errors that are specific to this API. One error example per
+section
+
+```http
+???
+```
+
+### Retrieve the active Import Session
+
+<ApiOperation method="get" url="/api/v1/identity-sources/${identitySourceId}/sessions" />
+
+Fetches the active Import Session for an identity source. An Import Session with a `CREATED` or `TRIGGERED` status is considered active.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `identitySourceId`  | String | The ID obtained from creating a Customer Identity Source integration in Okta |
+
+#### Response body
+
+A list of active [Import Session objects](#import-session-object)
+
+#### Use example
+
+This request...:
+
+##### Request
+
+```bash
+curl -L -X GET 'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions' \
+-H 'Authorization: SSWS ${apiKey}' \
+-H 'Accept: application/json' \
+-H 'Content-Type: application/json'  
+```
+
+##### Response
+
+```json
+[
+   {
+       "id": "uij4bjiw3eY00uuhR0g7",
+       "identitySourceId": "0oa4bjizmkh7KAJau0g7",
+       "status": "CREATED",
+       "importType": "INCREMENTAL"
+   }
+]
+ 
+```
+
+##### Error response example 
+
+Here you can show any errors that are specific to this API. One error example per section
+
+```http
+Hello error
+```
+
+### Trigger an Import Session
+
+<ApiOperation method="put" url="/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}/start-import" />
+
+Triggers the import process of the user data in an Import Session into Okta.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `identitySourceId`  | String | The ID obtained from creating a Customer Identity Source integration in Okta |
+| `sessionId`  | String | The ID for the Import Session to process |
+
+#### Response body
+
+Returns ???
+
+#### Possible errors
+
+??? This section is optional
+
+Here you can provide a list of error IDs and their descriptions that are specific to this API.
+
+| Error ID    | Description    |
```

Comment:
Will go over all the Error IDs with you.  I am not sure how extensive this section should be and will take your guidance during the call.